mqtt_as/interfaces/wlan/esp32.py

```diff
@@ -0,0 +1,41 @@
+from .. import BaseInterface
+import network
+import uasyncio as asyncio
+
+
+class WLAN(BaseInterface):
+    def __init__(self, ssid, wifi_pw):
+        super().__init__()
+        self.DEBUG = False
+        # https://forum.micropython.org/viewtopic.php?f=16&t=3608&p=20942#p20942
+        self.BUSY_ERRORS += [118, 119]  # Add in weird ESP32 errors
+        self._ssid = ssid
+        self._wifi_pw = wifi_pw
```

Comment:
how about ``self._pass``

Reply:
I was thinking about self._pw but kept it like that for now for better recognizability since it was named this way in mqtt_as. Once @peterhinch is happy with the changes, I'll shorten this variable. In this context it is very obvious what it does, so no need for longer names.